site/ko/beta/guide/distribute_strategy.ipynb

```diff
@@ -0,0 +1,975 @@
+        "\n",
+        "여기에 추가 성능 최적화도 구현하고 있습니다. 예를 들어 작은 텐서들의 여러 올 리듀스 작업을 큰 텐서들의 더 적은 올 리듀스 작업으로 바꾸는 정적 최적화 기능이 있습니다. 뿐만아니라 플러그인 구조를 갖도록 설계하였습니다. 따라서 추후에는 사용자가 자신의 하드웨어에 더 최적화된 알고리즘을 사용할 수도 있을 것입니다. 참고로 이 수집 연산은 올 리듀스 외에 브로드캐스트(broadcast)나 전체 수집(all-gather)도 구현하고 있습니다.\n",
+        "\n",
+        "`MultiWorkerMirroredStrategy`를 만드는 가장 쉬운 방법은 다음과 같습니다."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "m3a_6ebbEjre"
+      },
+      "outputs": [],
+      "source": [
+        "multiworker_strategy = tf.distribute.experimental.MultiWorkerMirroredStrategy()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "bt94JBvhEr4s"
+      },
+      "source": [
+        "`MultiWorkerMirroredStrategy`와 함께 사용할 수집 연산은 현재 두 가지 구현 중에 골라야 합니다. `CollectiveCommunication.RING`는 통신을 위해 gRPC를 사용하여 환형(ring) 수집 구조를 구현한 것입니다. `CollectiveCommunication.NCCL`는 [Nvidia의 NCCL](https://developer.nvidia.com/nccl)을 사용하여 구현한 것입니다. `CollectiveCommunication.AUTO`로 설정하면 런타임이 알아서 구현을 고릅니다. 최적의 수집 연산 구현 방식은 GPU의 수와 종류, 클러스터의 네트워크 연결 등에 따라 다를 수 있습니다. 다음과 같이 지정하면 됩니다."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "QGX_QAEtFQSv"
+      },
+      "outputs": [],
+      "source": [
+        "multiworker_strategy = tf.distribute.experimental.MultiWorkerMirroredStrategy(\n",
+        "    tf.distribute.experimental.CollectiveCommunication.NCCL)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "0JiImlw3F77E"
+      },
+      "source": [
+        "여러 GPU를 사용하는 것과 비교해서 여러 워커를 사용하는 것의 가장 큰 차이점은 역시 여러 워커에 대한 설정 부분입니다. 클러스터를 구성하는 각 워커에 \"TF_CONFIG\" 환경변수를 사용하여 클러스터 설정을 하는 것이 텐서플로의 표준적인 방법입니다. [아래쪽 \"TF_CONFIG\"](#TF_CONFIG) 항목에서 어떻게 하는지 자세히 살펴보겠습니다."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "E20tG21LFfv1"
+      },
+      "source": [
+        "Note: 이 전략은 아직 개선 중이고 더 많은 경우에 쓸 수 있도록 만들고 있기 때문에, [`실험 기능`](https://www.tensorflow.org/guide/version_compat#what_is_not_covered)으로 지원됩니다. 따라서 나중에 API가 바뀔 수 있음에 유념하십시오."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "kPEBCMzsGaO5"
+      },
+      "source": [
+        "### TPUStrategy\n",
+        "`tf.distribute.experimental.TPUStrategy`은 텐서플로 훈련을 텐서처리장치(Tensor Processing Unit, TPU)에서 하는 것입니다. TPU는 구글의 특별한 ASIC로써, 기계 학습 작업을 극적으로 가속하기 위하여 설계되었습니다. TPU는 구글 코랩, [Tensorflow Research Cloud](https://www.tensorflow.org/tfrc), [Google Compute Engine](https://cloud.google.com/tpu)에서 사용할 수 있습니다.\n",
```

Comment:
The subject of first sentence is a kind of strategy, and I felt that '수행합니다' is not appropriate as a verb for it, because subject of '수행합니다' is "TPU" not strategy. So I changed to "수행하는 전략입니다". Thanks for pointing out :)